Realm/Realm.Weaver/Analytics/Metric.cs

```diff
@@ -55,83 +66,132 @@ public static class Framework
             public const string Xamarin = "Xamarin";
         }
 
-        public static readonly Dictionary<string, string> UserEnvironment = new Dictionary<string, string>()
+        public static class Environment
```

Comment:
Replaced the dict with a class of constants since we weren't automating anything about the dictionary population like we do with the features.